services/bots/src/github-webhook/handlers/debug_info.ts

```diff
@@ -0,0 +1,23 @@
+import { IssuesLabeledEvent } from '@octokit/webhooks-types';
+import { EventType, HomeAssistantRepository } from '../github-webhook.const';
+import { WebhookContext } from '../github-webhook.model';
+import { BaseWebhookHandler } from './base';
+
+const COMMENTS = {
+    'cpu': ' - Start a new issue, choose profiler as the integration\n - Include a callgrind file from the <https://www.home-assistant.io/integrations/profiler/> `profiler.start` service\n - Include a `py-spy` from <https://community.home-assistant.io/t/instructions-to-install-py-spy-on-haos/480473>. (be sure to zip them up before posting as GitHub mutates svg files)\n - If memory is leaking, include 4 hours of RAW logs (`Settings -> System -> Logs -> Download`) from the <https://www.home-assistant.io/integrations/profiler/> `profiler.start_log_objects` service with the default settings'
+}
+
+export class DebugInfo extends BaseWebhookHandler {
+    public allowedEventTypes = [EventType.ISSUES_LABELED];
+    public allowedRepositories = [HomeAssistantRepository.CORE];
+
+    async handle(context: WebhookContext<IssuesLabeledEvent>) {
+        if (!context.payload.label && Object.values(COMMENTS).indexOf(context.payload.label.toString()) > -1) {
+            return;
+        }
+        context.scheduleIssueComment({
+            handler: 'DebugInfo',
+            comment: COMMENTS[context.payload.label],
```

Comment:
`context.payload.label` is an object, `context.payload.label.name` needs to be used.
<https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=labeled#issues>

Reply:
```suggestion
            comment,
```